spec/system/case_contacts/new_spec.rb

```diff
@@ -128,8 +128,8 @@
       expect(page).to have_text("Read more")
       expect(page).to have_text(expected_text)
 
-      sleep(2)
-      click_on "Close" # close thank-you modal
+      # sleep(2)
```

Comment:
This is probably a problem?

Reply:
Sleep was not required here. I was unsure why this was put in the first place, so I commented it instead of completely removing it.